ibis/backends/mysql/__init__.py

```diff
@@ -227,8 +226,11 @@ def get_schema(
         ).sql(self.dialect)
 
         with self.begin() as cur:
-            cur.execute(sge.Describe(this=table).sql(self.dialect))
-            result = cur.fetchall()
+            try:
+                cur.execute(sge.Describe(this=table).sql(self.dialect))
+                result = cur.fetchall()
+            except Exception:  # noqa: BLE001
```

Comment:
I don't like this, we could modify TableNotFound to also include the message of the original exception raised. Unclear weather that's the path or we are happy with this being raised in Ibis land.

Reply:
This seems like the wrong approach here. Instead of catching the exception, we should check if `result` is empty and if it is, raise `TableNotFound`. Otherwise we're potentially hiding a slew of bugs as `TableNotFound`. Anything from incorrect sqlglot code generation to transient errors from the server.

Reply:
I tried that first, but it fails before in here, so I don't get to do the `fetchall()`:
```
-> cur.execute(sge.Describe(this=table).sql(self.dialect))
(Pdb) pp cur.execute(sge.Describe(this=table).sql(self.dialect))
*** pymysql.err.ProgrammingError: (1146, "Table 'ibis_testing.foo' doesn't exist")
```